README.md

```diff
@@ -799,10 +799,10 @@ export TMPDIR=${TMPDIR:-/tmp}
 deno test \
  --allow-net \
  --allow-read \
- --allow-env=SRCROOT,GITHUB_TOKEN,TMPDIR,TEA_PREFIX \
+ --allow-env=SRCROOT,GITHUB_TOKEN,TMPDIR,TEA_PREFIX,MUGGLE,MAGIC,JSON,NUMPTY \
  --allow-run \
  --import-map=$SRCROOT/import-map.json \
- --allow-write="$TMPDIR" \
+ --allow-write \
```

Comment:
why?

Reply:
Because the test I have created need write access to `~/.tea/tea.xyz/var/www/https/dist.tea.xyz/` directory for writing mtime and etag files

Reply:
So set the new `--allow-write` to `--allow-write="$TMPDIR:$TEA_PREFIX/tea.xyz/var/www".

We don’t allow EVERYTHING just because one new thing needs more, you keep things as restrictive as possible.

Reply:
Frankly we should change TEA_PREFIX for the tests anyway.

Reply:
Comma, I think, not colon. You've spent too much time on the POSIX work.

Reply:
Fixed it, It is working on local but tests are getting failed. 

Seems like this is another issue with `$SRCROOT`. Tests that were working fine earlier are now failing :(